### PR TITLE
Added cp to notifications

### PR DESCRIPTION
**Pull request by C0DYcc**
For shinies that are common, whether it's an event that is spawning lots of the same pokemon or a nest, I don't like to catch them if they aren't a high level. So I added cp to the discord notification to help me decide if I want to log in and catch that shiny.

### CODE REVIEW

File: shinywatcher.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def get_cache():
     return open("cache.txt", "r").read().splitlines()
 
 def check_shinies():
-    cursor.execute(f"SELECT encounter_id, pokemon_id, disappear_time, individual_attack, individual_defense, individual_stamina, longitude, latitude, t.worker FROM pokemon LEFT JOIN trs_stats_detect_raw t ON encounter_id = CAST(t.type_id AS UNSIGNED INTEGER) WHERE disappear_time > utc_timestamp() AND t.is_shiny = 1 {worker_filter} ORDER BY pokemon_id DESC, disappear_time DESC")
+    cursor.execute(f"SELECT encounter_id, pokemon_id, disappear_time, individual_attack, individual_defense, individual_stamina, cp, longitude, latitude, t.worker FROM pokemon LEFT JOIN trs_stats_detect_raw t ON encounter_id = CAST(t.type_id AS UNSIGNED INTEGER) WHERE disappear_time > utc_timestamp() AND t.is_shiny = 1 {worker_filter} ORDER BY pokemon_id DESC, disappear_time DESC")
     results = cursor.fetchall()
     for enc_id, mon_id, etime, atk, defe, sta, lon, lat, worker in results:
```
Comment (ccev) [inline shinywatcher.py]: need to put cp in the for loop

File: shinywatcher.py [hunk 2]
```diff
@@ -80,7 +80,7 @@ def check_shinies():
             data = {
                 "username": mon_name,
                 "avatar_url": mon_img,
-                "content": f"**{mon_name}** ({iv}%) until **{end}** ({timeleft[0]}m {timeleft[1]}s)\n{worker} ({email})",
+                "content": f"**{mon_name}** ({iv}% cp{cp}) until **{end}** ({timeleft[0]}m {timeleft[1]}s)\n{worker} ({email})",
```
Comment (ccev) [inline shinywatcher.py:83]: maybe change this to {cp} CP

**Review by ccev — COMMENTED** [review]
good idea, wouldn't work like this i think. i don't even have this running currently so i can't check

**Comment by C0DYcc** [comment]
I did the cp{cp} just because that's how it is in game, I had it your way to begin with but noticed it was different in game. Not sure it's needed in the loop though, I've been running this for a couple weeks with just how I modified it with no issues.
EDIT: Damn you're right, I did that on mine but forgot to do that on the PR, whoops. I'm not too familiar with git but I double checked my file and the cp in the for loops is the only other thing I did.

**Comment by ccev** [comment]
I decided to put the Mon's Level in instead since it tells you more about the mon. Thank you anyway for PRing it. Much appreciated!!

**Pull request closed without merging**